queue/src/fiskaltrust.Middleware.Storage.AzureTableStorage/DatabaseMigrator.cs

```diff
@@ -37,12 +42,24 @@ public async Task MigrateAsync()
             if (await MigrationTableExists())
             {
                 var currentMigration = await GetCurrentMigrationAsync();
+                MigratedFrom1_2 = await GetMigratedFrom1_2Async();
                 await ExecuteMigrationsAsync(_migrations.Where(x => x.Version > currentMigration));
             }
             else
             {
                 _logger.LogInformation("Database tables were not yet created, executing all {MigrationCount} migrations.", _migrations.Length);
                 await _tableServiceClient.CreateTableIfNotExistsAsync(GetMigrationTableName());
+
+                var cashboxTableName = $"x{_queueConfiguration.QueueId.ToString().Replace("-", "")}{AzureTableStorageCashBoxRepository.TABLE_NAME}";
+                if (await _tableServiceClient.QueryAsync(t => t.Name == cashboxTableName).AnyAsync())
+                {
+                    var cashBoxTableClient = _tableServiceClient.GetTableClient(cashboxTableName);
+                    var cashBoxes = await cashBoxTableClient.QueryAsync<TableEntity>().FirstOrDefaultAsync();
+                    if (cashBoxes is not null)
+                    {
+                        MigratedFrom1_2 = true;
```

Comment:
Just to confirm. We set this to true, because if there is no migration table yet, but we are still finding a cashboxes table, it is very likely that this is a migration.

Reply:
yep 👌 It' the best way I could think of to detect that